people/john-williams.md

```diff
@@ -0,0 +1,6 @@
+John Williams
+
+![](photos/john_profile.jpg)
```

Comment:
I am sure there's another John in the class. Call this john-williams.jpg to match the md file, please.